plugins/gradle-plugin/src/main/kotlin/com/android/designcompose/gradle/Plugin.kt

```diff
@@ -46,40 +46,44 @@ import org.gradle.configurationcache.extensions.capitalized
  * @constructor Create empty Figma token plugin
  */
 class Plugin : Plugin<Project> {
+
+    private lateinit var pluginExtension: PluginExtension
     override fun apply(project: Project) {
-        project.plugins.withType(com.android.build.gradle.AppPlugin::class.java) {
+        pluginExtension = project.extensions.create("designcompose", PluginExtension::class.java)
+        pluginExtension.initialize(project)
+
+        project.pluginManager.withPlugin("com.android.application") {
```

Comment:
[Gradle doc](https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:plugins) says that it's preferred to use pluginManager instead of plugins